src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -185,8 +185,18 @@ public void postCommentPerIssue(String project, String repository, String pullRe
     }
   }
   
-  public StashCredentials getCredentials(){
-    return new StashCredentials(config.getStashLogin(), config.getStashPassword());
+  public StashCredentials getCredentials() throws StashConfigurationException {
```

Comment:
Should we not catch the exception in the same method?
For readability, I would tend to prefer full name variables like passwordEnv